corehq/apps/registry/templates/registry/registry_list.html

```diff
@@ -0,0 +1,169 @@
+            <i class="fa fa-plus"></i>
+            {% trans "New Registry" %}
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-12 col-md-9">
+      <!-- ko foreach: ownedRegistries -->
+      <div class="panel panel-default show-focus">
+        <div class="panel-heading clickable">
+          <span data-bind="text: name"></span>
+          <div class="pull-right">
+            <span class="label label-primary">
+              <i class="fa fa-user"></i>
+              {% trans "Owner" %}
+            </span>
+            <button type="button" class="btn btn-default btn-xs" data-bind="click: showDeleteModal">
+              <i class="fa fa-trash"></i>
+            </button>
+          </div>
+        </div>
+        <div class="panel-body">
+          <p data-bind="text: description"></p>
+          <div class="label label-plain" data-bind="ifnot: invitation_count">
+            {% trans "No project spaces invited to this registry" %}
+          </div>
+          <div class="label label-plain" data-bind="if: accepted_invitation_count">
+            <i class="fa fa-check-circle-o text-success"></i>
+            <span data-bind="text: acceptedText"></span>
+          </div>
+          <div class="label label-plain" data-bind="if: pending_invitation_count">
+            <i class="fa fa-clock-o text-info"></i>
+            <span data-bind="text: pendingText"></span>
+          </div>
+          <div class="label label-plain" data-bind="if: rejected_invitation_count">
+            <i class="fa fa-ban text-warning"></i>
+            <span data-bind="text: rejectedText"></span>
+          </div>
+          <button type="button" class="btn btn-default btn-xs" data-bind="click: inviteProject">
+            <i class="fa fa-plus"></i>
+            {% trans "Invite Project" %}
+          </button>
+        </div>
+      </div>
+      <!-- /ko -->
+
+    <!-- ko foreach: invitedRegistries -->
+      <div class="panel panel-default show-focus">
+        <div class="panel-heading clickable">
+          <span data-bind="text: name"></span>
+          <div class="pull-right">
+            <span class="label label-warning" data-bind="if: invitation.status === 'rejected'">
+              <i class="fa fa-user"></i>
+              {% trans "Rejected" %}
+            </span>
+            <span class="label label-success" data-bind="if: invitation.status === 'accepted'">
+              <i class="fa fa-user"></i>
+              {% trans "Participant" %}
+            </span>
+            <span class="label label-info" data-bind="if: invitation.status === 'pending'">
+              <i class="fa fa-user"></i>
+              {% trans "Pending" %}
+            </span>
+          </div>
+        </div>
+        <div class="panel-body">
+          <p data-bind="text: description"></p>
+          <p><small><b>{% trans "Owner" %}:</b><b data-bind="text: domain_name"></b></small></p>
```

Comment:
Nitpick: `<p class="help-block">` would be a similar effect and wouldn't use the non-semantic small and b tags.